README.md

```diff
@@ -1,21 +1,18 @@
 # Block Site
 
-**Block Site** (formerly known as Stop Social Media) is a simple Chrome extension that improves your productivity by blocking the access to time-consuming websites (like during the working hours) as you specify in **Options**. **Block Site** is by default disabled and doesn't block any website until you say so in **Options**.
+**Block Site** (formerly known as Stop Social Media) is a simple Chrome extension that improves your productivity by blocking access to distracting websites as you specify.
 
-### Options
+### Usage
 
-To open Options, **right-click** on the **Block Site** icon in the toolbar (looks like a letter **B**) and choose **Options** from the menu.
-
-### Example
-
-The sites you decide to block to boost your productivity, is up to you. Here's an example:
+To configure, click on the **B** icon in the toolbar and select **Options** from the menu. Enter sites in the textbox, leaving a new line for each site, like this:
 
 ```
 facebook.com
 instagram.com
 youtube.com
+!music.youtube.com
 twitter.com
 reddit.com
```

Comment:
Adding an example for `pathname` would help. Using subreddits on `reddit.com` for this would be a great example. 
e.g. `!reddit.com/r/MachineLearning/`

Reply:
Great example! Let's have it. If I may suggest, let's modify the example just a slight bit, **without a trailing slash,** as:

```
!reddit.com/r/MachineLearning
```

Which now explains two things. Not only it shows it **allows** `reddit.com/r/MachineLearning`, but also it shows that trailing slash is **not required** to make this work (as reddit redirects it to `reddit.com/r/MachineLearning/`).